### ISSUE TITLE
[a11y] We don't mark the disabled objects as disabled

### ISSUE THREAD

**Issue by unlikelyzero** [nasa/openmct#7322]
#### Summary
The a11y tests picked up that we're not marking disabled objects with the html semantic for disabled. We're just marking as !important . 

According to https://sean-elliott.medium.com/a11y-tips-disabled-buttons-and-colour-contrast-f8824d5e9610, the a11y tooling will detect when an element is disabled and ignore color contrast violations. So this should be as simple as marking disabled objects as disabled with html semantics.

Known offenders:
- [ ] Create Button
- [ ] Edited Object in Tree
- [ ] Dropdown in Fault Management Select Options

#### Environment
* Open MCT Version: 3.3.0

#### Impact Check List
- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [x] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

#### Additional Information

<img width="225" alt="Screenshot 2023-12-20 at 9 36 00 PM" src="https://github.com/nasa/openmct/assets/12446869/e5f69345-93fd-4cc5-acfd-f5134968411d">

**Comment by charlesh88** [comment]
### Testing Notes
#### Action menu: 
1. Go to a Telemetry Table with some data. Ensure that no rows are selected.
2. Invoke the Action menu. Verify that "Export Marked Rows" and "Unmark All Rows" appear disabled and cannot be clicked.
![Screenshot 2024-01-04 at 3 36 45 PM](https://github.com/nasa/openmct/assets/1056412/5aa909a8-e9b1-460d-befc-49f81e45996e)
3. Inspect the menu with Dev Tools. On the Elements tab, verify that each `<li>` element is using `aria-disabled = "true"` and that the class definition doesn't include "disabled".
![Screenshot 2024-01-04 at 3 42 35 PM](https://github.com/nasa/openmct/assets/1056412/9651ff8c-8858-4350-bd87-065913002c09)
4. Select some rows in the table. Repeat step 2, but verify that the previously disabled menu items are now enabled and can be clicked.
![Screenshot 2024-01-04 at 3 37 33 PM](https://github.com/nasa/openmct/assets/1056412/37b11d93-9707-468e-92ba-79a7876fadb6)

#### Create button:
1. Edit any editable view.
2. Verify that the Create button appears disabled and can't be clicked.
3. Inspect the button with Dev Tools. On the Elements tab, verify that the button is using `aria-disabled = "true"` and that the class definition doesn't include "disabled".
![Screenshot 2024-01-04 at 4 00 20 PM](https://github.com/nasa/openmct/assets/1056412/6dd3c930-fb1b-4ef2-9631-5bb4d93e2a73)
4. Exit Edit mode. Verify the button appears enabled and can be clicked.

#### Tree items
NOTE: a purely stylistic change was made here to improve contrast of tree items currently being edited; a `disabled` property is not being used.
1. Navigate to an editable view, then edit it.
2. In the tree, verify that the currently navved and editing tree item appears with italic text, but does NOT appear dimmed back. 
![Screenshot 2024-01-04 at 4 05 43 PM](https://github.com/nasa/openmct/assets/1056412/8ecfec9f-ff0b-4e30-9c53-c37d796052e2)

**Comment by jvigliotta** [comment]
Verified Fixed in Testathon 1/31/24

**Comment by ozyx** [comment]
Verified Testathon 2/5/2024

Clicking on disabled menu items still closes the menu, though. We should change that
